integr8ly/installation
# INTLY-7206: Add apicurito upgrade playbook

## Additional Information
JIRA: https://issues.redhat.com/browse/INTLY-7206

## Verification Steps
This has been verified in the original PR, see for logs and verification: https://github.com/integr8ly/installation/pull/1349
